### PR TITLE
Update EEx example

### LINKED ISSUE #22: EEx examples outdated
The current examples do not work. 

We need to adapt the examples to match the behaviour of the current lib.

### PR DESCRIPTION
**Pull request by tomkonidas**
## 📖 Description

Fix the documentation for the EEx example.

The breaking changes where introduced by: https://github.com/Omerlo-Technologies/ex_prosemirror/pull/20

Fixes #22

## 📋 Type of change

- [X] This change requires a documentation update

### CODE REVIEW

**Review by onixus74 — APPROVED** [review]

**Review by Matsa59 — APPROVED** [review]

File: lib/ex_prosemirror.ex [hunk 1]
```diff
@@ -9,10 +9,31 @@ defmodule ExProsemirror do
...
+      title: [
+        marks: [:em],
+        blocks: [:h1]
+      ]
+    ]
+  ```
+
+  We will get the following results:
+
+  * The `title` input will be instantiated with block `h1` and will have the `em` mark
```
Comment (Matsa59) [inline lib/ex_prosemirror.ex:34]:
```suggestion
  * Input of type `:title` will be instantiated with block `h1` and will have the `em` mark
```

File: lib/ex_prosemirror.ex [hunk 2]
```diff
@@ -9,10 +9,31 @@ defmodule ExProsemirror do
...
+  * The `body` input will be instantiated with blocks `p` and `h2` and will have the `strong` mark
```
Comment (Matsa59) [inline lib/ex_prosemirror.ex:35]:
```suggestion
  * Input of type `:body` will be instantiated with blocks `p` and `h2` and will have the `strong` mark
```

File: lib/ex_prosemirror.ex [hunk 3]
```diff
@@ -9,10 +9,31 @@ defmodule ExProsemirror do
...
+
```
Comment (Matsa59) [inline lib/ex_prosemirror.ex:36]: Maybe you can write a usage sample 
E.g.

---

## Usages
```elixir
<%= prosemirror_input f, :title, type: :body ... %>
```

**Review by Matsa59 — APPROVED** [review]

File: lib/ex_prosemirror.ex (hunk 3 above)
Comment (tomkonidas) [reply lib/ex_prosemirror.ex:44]: hows that?

**Pull request merged**